spec/controllers/users/sessions_controller_spec.rb

```diff
@@ -271,12 +271,39 @@
   end
 
   describe '#new' do
-    it 'tracks page visit' do
-      stub_analytics
+    context 'with fully authenticated user' do
+      it 'redirects to the profile page' do
+        stub_sign_in
+        get :new
+
+        expect(response).to redirect_to profile_path
```

Comment:
Would it be worth it to also make sure that the user does not get signed out? For example, if you add `sign_out` after `redirect_to after_sign_in_path_for(current_user)` in the Sessions Controller, this test will still pass.